src/factories/ModuleFactory.sol

```diff
@@ -76,18 +79,21 @@ contract ModuleFactory is
     //--------------------------------------------------------------------------
     // Storage
 
+    /// @inheritdoc IModuleFactory
+    address public governanceContract; //@note changeable?
```

Comment:
If we do make it changeable, who would be authorized to do so? Seems to me that this should also be managed by the community multisig, otherwise its a backdoor.

Reply:
Through the planned upgradeability of the contract this can be added later, so i wont include it for now due to timeconstraints, but would be nice to have a decision on this @marvinkruse

Reply:
Oh sorry, just saw this now. It's fine to keep it like it is implemented now (no changeability). The governor is upgradeable, that's fine.